ci/install_deps.sh

```diff
@@ -13,10 +13,10 @@ import sys
 site.ENABLE_USER_SITE = '--user' in sys.argv[1:]
 $(cat ./ibis/setup.py)" > ./ibis/setup.py
 
-    python -m pip install --user -e ./ibis
+    python -m pip install --user ./ibis
 else
     python -m pip install --user ibis-framework=="$IBIS_VERSION"
 fi
 
-python -m pip install --user -e .
+python -m pip install --user ./ibis_bigquery
```

Comment:
The nested install (checking out `ibis` inside of `ibis_bigquery`) was causing duplicated results for the `ibis.backends.bigquery` entry point from `importlib.metadata.entry_points()`.